catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/SolrFilterDelegateTest.java

```diff
@@ -456,6 +456,87 @@ public void testDatePropertyIsBetween() {
     assertThat(temporalQuery.getQuery(), is(expectedQuery));
   }
 
+  @Test
```

Comment:
✏️ You could make a helper method to avoid repeating code in all these tests. Just pass the `AttributeFormat`, `lowerBoundary` and `upperBoundary` as parameters 
i.e. 
`public void testNumberIsBetweenLongs() {
  yourHelperMethod(AttributeFormat.Long, -100, 100);
}`